Add SDK ID API and messagebird signing key

mspensieri opened the pull request:
1. Add new API to retrieve the IDs of the default SDK integrations of
each type
2. Added `signingKey` when creating a `messagebird` integration

fxlemire commented on Smooch.postman_collection.json:
```diff
@@ -3538,6 +3538,35 @@
+						},
+						"description": "### For more information\n- API Docs: [https://docs.smooch.io/rest/#get-sdk-ids](https://docs.smooch.io/rest/#get-sdk-ids)"
+					},
+					"response": []
```
I see examples below have `response` filled, is that a place for an example of what the `response` is? If so maybe we could add it? 🤔🤷‍♂️

mspensieri replied:
How do I do that in postman?

mspensieri replied:
Figured it out, fixed I think

fxlemire replied:
Glad you did, cuz I would not have known how! 😂

fxlemire commented on Smooch.postman_collection.json:
```diff
@@ -5639,7 +5668,7 @@
 								"body": {
 									"mode": "raw",
-									"raw": "{\n\t\"type\": \"messagebird\",\n    \"accessKey\": \"<messagebirdAccessKey>\",\n    \"originator\": \"<messagebirdOriginator>\"\n}"
+									"raw": "{\n\t\"type\": \"messagebird\",\n    \"accessKey\": \"<messagebirdAccessKey>\",\n    \"originator\": \"<messagebirdOriginator>\",\n    \"signingKey\": \"<messagebirdSigningKey>\"\n}"
```
Should that also be added here as well?

https://github.com/smooch/smooch-postman-collection-public/pull/15/files#diff-4b081629b5ae173add1b20828fb6033bR5704

fxlemire commented on Smooch.postman_collection.json:
```diff
@@ -10912,11 +10941,14 @@
 								"key": "Content-Type",
-								"value": "application/json"
+								"value": "application/json",
+								"type": "text"
 							},
 							{
+								"warning": "This is a duplicate header and will be overridden by the Authorization header generated by Postman.",
 								"key": "Authorization",
-								"value": "Bearer {{JWT}}"
+								"value": "Bearer {{JWT}}",
+								"type": "text"
```
Is that intended?

mspensieri replied:
Dunno, came out with the export. I'll revert

mspensieri committed:
Add SDK ID API and messagebird signing key

1. Add new API to retrieve the IDs of the default SDK integrations of
each type
2. Added `signingKey` when creating a `messagebird` integration
